addons/note/models/note.py

```diff
@@ -56,10 +56,6 @@ def _get_default_stage_id(self):
         comodel_name='res.partner', string='Followers (Partners)',
         compute='_get_followers', search='_search_follower_partners',
         compute_sudo=True)
-    message_channel_ids = fields.Many2many(
-        comodel_name='mail.channel', string='Followers (Channels)',
-        compute='_get_followers', search='_search_follower_channels',
-        compute_sudo=True)
```

Comment:
Probably version number in note manifest to increase.

Reply:
Double check all modules actually, I think even mail version is currently not increased in current diff.

Reply:
Because it is already increased in this saas. For note, well I made Chs happy with not increasing it. And my scripts are already r+ :D .

Reply:
But wasn't the point of increasing this value so that we can run the migration script in master / rolling release scenario?
If we only increase it on saas release increasing the module versions it totally pointless.